### ISSUE TITLE
Crash report - Index 1 is out of bound

### ISSUE THREAD

**Issue by danielhopkins** [github/hub#1171]
Error (*errors.errorString): `Index 1 is out of bound`

Stack:

```
goroutine 1 [running]:
github.com/github/hub/commands.(*Args).InsertParam(0xc2080112d0, 0x1, 0xc20829dd38, 0x4, 0x4)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/args.go:95 +0x170
github.com/github/hub/commands.transformCheckoutArgs(0xc2080112d0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/checkout.go:102 +0xd5b
github.com/github/hub/commands.checkout(0x6826a0, 0xc2080112d0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/checkout.go:36 +0x40
github.com/github/hub/commands.(*Command).Call(0x6826a0, 0xc2080112d0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/commands.go:46 +0x189
github.com/github/hub/commands.(*Runner).Call(0xc20802b120, 0x6826a0, 0xc2080112d0, 0x0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/runner.go:88 +0x54
github.com/github/hub/commands.(*Runner).Execute(0xc20802b120, 0x0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/runner.go:80 +0x1e4
main.main()
	/private/tmp/hub-lKNcwz/main.go:12 +0x40

```

Runtime:

```
GOOS: darwin
GOARCH: amd64
```

**Comment by mislav** [comment]
Hi thanks for reporting! And ouch for the crash 😓  Can you tell us which arguments were you supplying to `hub checkout` and which `hub version` do you have? Thanks

**Comment by danielhopkins** [comment]
It was `hub checkout -t "PR_FROM_PRIVATE_REPO"`

**Comment by danielhopkins** [comment]
`git version 2.7.1
hub version 2.2.0-rc1-g2c12de3`

**Comment by mislav** [comment]
@danielhopkins There's a good chance that this was fixed already. Could you upgrade to v2.2.3 and try again?

**Comment by mislav** [comment]
I'm going to guess that this was fixed already in newer releases of 2.2.x. 2884da15b2a73097d5b51dc40aa047e1ef2ec56b
